2_isolation/README.md

```diff
@@ -28,3 +35,11 @@ $ python pit.py
 1. [Heuristics Analysis](heuristic_analysis.md)
 
 2. [Research Review](research_review.md)
+
+---
+
+## Screenshot
+
+![alt tag](https://raw.githubusercontent.com/ltfschoen/nd889/master/2_isolation/screenshots/screenshot.png)
```

Comment:
Awesome, a screenshot on the main README.md is good idea. Maybe change to the img folder so we dont have two image folders. Also I have made a screenshot with the same terminal theme on my machine and ill upload it over the top once iv merged your PR so the colours match.